ably/ably-ui
# Add icons

Jira Ticket Link / Motivation
----



Sync with Figma icons.

Note, this does not cover breaking changes to some of them, i.e., renames and changes of section.


Summary of changes
----


See commits for details.

How do you manually test this?
----



See review app.

Reviewer Tasks (optional)
----




Merge/Deploy Checklist
----




Frontend Checklist
----
